deps/Makefile

```diff
@@ -263,7 +263,7 @@ endif
 else # NO_GIT
 $2_SRC_DIR = $1-$$($2_BRANCH)
```

Comment:
Ok. For keeping them updated I may want to reinstate what this PR originally was doing, which is building with `NO_GIT` on Travis and AppVeyor.